cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -110,6 +110,7 @@ controller, and serviceaccounts controller.`,
 	return cmd
 }
 
+// get the synchronization Period
```

Comment:
Comments for public methods should be in godoc style. Additionally, IMO, comments that explain *why* something is done are many times more useful than those that just state *what* is being done, and comments that explain what is being done are in turn more useful than those that just restate the name of the function in other words, which is what your proposed comment is doing.

For example, to put a useful comment here, I recommend something like:

```go
// ResyncPeriod returns a function which generates a duration each time it is 
// invoked; this is so that multiple controllers don't get into lock-step and all 
// hammer the apiserver with list requests simultaneously.
```